src/computer.cpp

```diff
@@ -502,7 +505,7 @@ void computer::activate_function(computer_action action, char ch)
         }
         if(query_yn(_("Confirm nuclear missile launch."))) {
             add_msg(m_info, _("Nuclear missile launched!"));
-            options.clear();//Remove the option to fire another missle.
+            options.clear();//Remove the option to fire another missile.
```

Comment:
It's nice to know that I'm not the only one who fixes misspelled comments. Good stuff, @illi-kun!